src/patternfly/components/AppLauncher/app-launcher-menu-item-description.hbs

```diff
@@ -0,0 +1,6 @@
+<p class="pf-c-app-launcher__menu-item-description{{#if app-launcher-menu-item-description--modifier}} {{app-launcher-menu-item-description--modifier}}{{/if}}"
```

Comment:
Do you know this should be a `<p>` for sure? Users wouldn't be able to use non-phrasing content here with a `<p>`, like a `<ul/ol/dl>`, `<div>`, etc.